openedx_cas/apps.py

```diff
@@ -11,3 +11,12 @@ class OpenedxCasConfig(AppConfig):
     """
 
     name = 'openedx_cas'
+
+    plugin_app = {
+        'settings_config': {
+            'lms.djangoapp': {
+                'test': {'relative_path': 'settings.test'},
+                'common': {'relative_path': 'settings.common'}
+            }
+        },
+    }
```

Comment:
New line missing. This would be nice in another PR; leave this one just for cas flow changes. Meaning creating a new PR with this change, the change of removing test_models, and setup.py changes. We can merge that PR right away